internal/cri/opts/spec_opts.go

```diff
@@ -118,6 +118,31 @@ func WithAnnotation(k, v string) oci.SpecOpts {
 	}
 }
 
+// WithWindowsAffinityCPUs sets the CPU affinity values in runtime spec for windows.
+func WithWindowsAffinityCPUs(config *runtime.WindowsContainerConfig) oci.SpecOpts {
+	return func(_ context.Context, _ oci.Client, c *containers.Container, s *runtimespec.Spec) error {
+		// If Windows Resources or AffinityCpus are not set, return nil
+		if s.Windows == nil || s.Windows.Resources == nil ||
```

Comment:
To be sure, we never want to add this if s.Windows.Resources is not set?
Just curious why we wouldn't fill it if its nil.